content/lessons/testbed-lesson.md

```diff
@@ -1,22 +1,17 @@
 ---
 title: Testbed Lesson
 description: Brief, 1-2 sentence description of lesson. Shows under search results and elsewhere.
+date: 2112-12-24
 video: spUNpyF58BY
 credits:
   - Lesson and figures by Grant Sanderson
   - Translated to blog format by James Schloss
-tags:
-  - interactive
 ---
 
 Markdown content.
 One sentence per line.
 
-<!-- html comment (will appear in the rendered page's source code) -->
-
-{% comment %}
```

Comment:
Yeah I took those out because I couldn't think of a case where you guys would want to show a comment in the site's source code. Is there a real use case? Seems like it'd be easy to add later if you ever needed it.

Reply:
Easy to add later. I think the use case would be keeping notes on lessons for future selves and other authors.